docs/modules/ROOT/pages/concepts/gauges.adoc

```diff
@@ -42,7 +42,7 @@ Note that, in this form, unlike other meter types, you do not get a reference to
 
 This pattern should be less common than the `DoubleFunction` form. Remember that frequent setting of the observed `Number` results in a lot of intermediate values that never get published. Only the _instantaneous value_ of the gauge at publish time is ever sent to the monitoring system.
 
-WARNING: Attempting to construct a gauge with a primitive number or one of its `java.lang` object forms is always incorrect. These numbers are immutable. Thus, the gauge cannot ever be changed. Attempting to "`re-register`" the gauge with a different number does not work, as the registry maintains only one meter for each unique combination of name and tags.
+WARNING: Attempting to construct a gauge with a primitive number or one of its `java.lang` object forms is always incorrect. These numbers are immutable. Thus, the gauge cannot ever be changed. Attempting to "re-register" the gauge with a different number does not work, as the registry maintains only one meter for each unique combination of name and tags. "Re-registering" a gauge can happen indirectly for example as the result of a `MeterFilter` modifying the name and/or the tags of two different gauges so that they will be the same after the filter is applied.
```

Comment:
Not sure if text should be added for this here, but this problem with gauges seems like another reason to user Counter meters instead of gauges if possible.   The docs to nudge twoards using counters instead of gauges another place.

Reply:
I wouldn't add text to that effect because already if instead of a gauge you could use a counter for the metric, you should. Adding text about this as a reason to use a counter instead of a gauge might cause readers to infer if they don't have this issue it is okay to use a gauge as a counter.